index.js

```diff
@@ -155,13 +155,13 @@ module.exports = {
         });
       },
       linkifyTweetMentions: function (text) {
-        return text.replace(/@[^\W]+/g, function (user) {
+        return text.replace(/@[\w]+/g, function (user) {
```

Comment:
Sorry, one last thing (:

`\w` doesn't need to be inside `[]`, which is used mainly to enable ranges like `[a-z]`. `\w` is already understood to match any word character.

So you can drop the `[]`.

Last nitpick!

Reply:
Good call!